chrome/android/java/src/org/chromium/chrome/browser/init/AsyncInitializationActivity.java

```diff
@@ -280,10 +282,12 @@ public void onStartupFailure() {
     protected final void onCreate(Bundle savedInstanceState) {
         // Hide the activity if castanets is enabled and runs as renderer process,
         if (OffloadingUtils.IsCastanets()) {
```

Comment:
Very good.
I think OffloadingUtils.IsCastanets() isn't necessary anymore.
Could you remove this condition?

Reply:
Done.